src/Models/Behaviors/HasMedias.php

```diff
@@ -15,6 +16,16 @@ trait HasMedias
         'crop_h',
     ];
 
+    public static function bootHasMedias(): void
```

Comment:
@haringsrob this totally works and is standard in Laravel however I'm wondering if this is the right location in Twill, knowing we have repository level hooks. What do you think?

Reply:
This is just here to avoid media from being unable to delete when it is force deleted using some other call. (outside of twill).

If I run $myModel->forceDelete(); via my own API or controller. I might not use the repository to do so.